library/openshift_provision.py

```diff
@@ -230,9 +230,12 @@ def compare_set_list(path, src, dst):
     return list(compare_values(['/' + field], current, config))
 
 def set_dict_defaults(d, default):
-    for k, v in default.items():
-        if k not in d:
-            d[k] = v
+    if d is None:
+        d = default.items()
+    else: 
+        for k, v in default.items():
+            if k not in d:
+                d[k] = v
```

Comment:
This block of code should fix https://github.com/gnuthought/ansible-role-openshift-provision/issues/53